### PR TITLE
remove unnecessary steps

### PR DESCRIPTION
**Pull request by MaratMussabekov**
#5417

### CODE REVIEW

File: mdop/mbam-v25/upgrading-to-mbam-25-sp1-from-mbam-25.md [hunk 1]
```diff
@@ -29,21 +29,17 @@ Verify you have a current documentation of your MBAM environment, including all
...
 #### Steps to upgrade the MBAM Server (Running MBAM and IIS)
 1. Using the MBAM Configurator; remove the Admin and Self Service Portals from  the IIS server
 2. Install MBAM 2.5 SP1
 3. Do not configure it at this time  
-4. Install the May 2019 Rollup on the IIS server(https://www.microsoft.com/download/details.aspx?id=58345)
-5. Using the MBAM Configurator; re-add the Admin and Self Service Portals to the IIS server 
-6. This will configure the sites using the latest MBAM code from the May 2019 Rollup
-7. Open an elevated command prompt, Type: **IISRESET** and Hit Enter.
+4. Using the MBAM Configurator; re-add the Admin and Self Service Portals to the IIS server 
+5 Open an elevated command prompt, Type: **IISRESET** and Hit Enter.
```
Comment (illfated) [inline mdop/mbam-v25/upgrading-to-mbam-25-sp1-from-mbam-25.md]: minor punctuation typo
```suggestion
5. Open an elevated command prompt, Type: **IISRESET** and Hit Enter.
```

**Comment by mypil** [comment]
@JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks.

File: mdop/mbam-v25/upgrading-to-mbam-25-sp1-from-mbam-25.md [hunk 2]
```diff
@@ -29,21 +29,17 @@ Verify you have a current documentation of your MBAM environment, including all
...
-4. Install the May 2019 Rollup: https://www.microsoft.com/download/details.aspx?id=58345
-5. Using the MBAM Configurator; re-add the Reports role
-6. This will configure the SSRS connection using the latest MBAM code from the rollup 
-7. Using the MBAM Configurator; re-add the SQL Database role on the SQL Server.
-8. At the end, you will be warned that the DBs already exist and  weren’t created, but this is expected.
-9. This process updates the existing databases to the current version  being installed                  
+4. Using the MBAM Configurator; re-add the Reports role
+5. Using the MBAM Configurator; re-add the SQL Database role on the SQL Server.
+6. At the end, you will be warned that the DBs already exist and  weren’t created, but this is expected.
+7. This process updates the existing databases to the current version  being installed
```
Comment (JohanFreelancer9) [inline mdop/mbam-v25/upgrading-to-mbam-25-sp1-from-mbam-25.md]:
```suggestion
7. This process updates the existing databases to the current version being installed.              
```

File: mdop/mbam-v25/upgrading-to-mbam-25-sp1-from-mbam-25.md [hunk 3]
```diff
@@ -29,21 +29,17 @@ Verify you have a current documentation of your MBAM environment, including all
    Note: You will not see an option to remove the Databases; this is expected.  
 2. Install 2.5 SP1 (Located with MDOP - Microsoft Desktop Optimization Pack 2015 from the Volume Licensing Service Center site:  <https://www.microsoft.com/Licensing/servicecenter/default.aspx>
 3. Do not configure it at this time 
-4. Install the May 2019 Rollup: https://www.microsoft.com/download/details.aspx?id=58345
-5. Using the MBAM Configurator; re-add the Reports role
-6. This will configure the SSRS connection using the latest MBAM code from the rollup 
-7. Using the MBAM Configurator; re-add the SQL Database role on the SQL Server.
-8. At the end, you will be warned that the DBs already exist and  weren’t created, but this is expected.
-9. This process updates the existing databases to the current version  being installed                  
+4. Using the MBAM Configurator; re-add the Reports role
```
Comment (JohanFreelancer9) [inline mdop/mbam-v25/upgrading-to-mbam-25-sp1-from-mbam-25.md]:
```suggestion
4. Using the MBAM Configurator; re-add the Reports role.
```

File: mdop/mbam-v25/upgrading-to-mbam-25-sp1-from-mbam-25.md [hunk 4]
```diff
@@ -29,21 +29,17 @@ Verify you have a current documentation of your MBAM environment, including all
...
+5. Using the MBAM Configurator; re-add the SQL Database role on the SQL Server.
+6. At the end, you will be warned that the DBs already exist and  weren’t created, but this is expected.
+7. This process updates the existing databases to the current version  being installed                  
 
 #### Steps to upgrade the MBAM Server (Running MBAM and IIS)
 1. Using the MBAM Configurator; remove the Admin and Self Service Portals from  the IIS server
```
Comment (JohanFreelancer9) [inline mdop/mbam-v25/upgrading-to-mbam-25-sp1-from-mbam-25.md]:
```suggestion
1. Using the MBAM Configurator; remove the Admin and Self Service Portals from  the IIS server.
```

File: mdop/mbam-v25/upgrading-to-mbam-25-sp1-from-mbam-25.md [hunk 5]
```diff
@@ -29,21 +29,17 @@ Verify you have a current documentation of your MBAM environment, including all
...
 2. Install MBAM 2.5 SP1
```
Comment (JohanFreelancer9) [inline mdop/mbam-v25/upgrading-to-mbam-25-sp1-from-mbam-25.md]:
```suggestion
2. Install MBAM 2.5 SP1.
```

File: mdop/mbam-v25/upgrading-to-mbam-25-sp1-from-mbam-25.md [hunk 6]
```diff
@@ -29,21 +29,17 @@ Verify you have a current documentation of your MBAM environment, including all
...
 3. Do not configure it at this time
```
Comment (JohanFreelancer9) [inline mdop/mbam-v25/upgrading-to-mbam-25-sp1-from-mbam-25.md]:
```suggestion
3. Do not configure it at this time.
```

File: mdop/mbam-v25/upgrading-to-mbam-25-sp1-from-mbam-25.md [hunk 7]
```diff
@@ -29,21 +29,17 @@ Verify you have a current documentation of your MBAM environment, including all
...
 
 #### Steps to upgrade the MBAM Server (Running MBAM and IIS)
 1. Using the MBAM Configurator; remove the Admin and Self Service Portals from  the IIS server
 2. Install MBAM 2.5 SP1
 3. Do not configure it at this time  
-4. Install the May 2019 Rollup on the IIS server(https://www.microsoft.com/download/details.aspx?id=58345)
-5. Using the MBAM Configurator; re-add the Admin and Self Service Portals to the IIS server 
-6. This will configure the sites using the latest MBAM code from the May 2019 Rollup
-7. Open an elevated command prompt, Type: **IISRESET** and Hit Enter.
+4. Using the MBAM Configurator; re-add the Admin and Self Service Portals to the IIS server
```
Comment (JohanFreelancer9) [inline mdop/mbam-v25/upgrading-to-mbam-25-sp1-from-mbam-25.md]:
```suggestion
4. Using the MBAM Configurator; re-add the Admin and Self Service Portals to the IIS server.
```

File: mdop/mbam-v25/upgrading-to-mbam-25-sp1-from-mbam-25.md [hunk 8]
```diff
@@ -29,21 +29,17 @@ Verify you have a current documentation of your MBAM environment, including all
...
 #### Steps to upgrade the MBAM Server (Running MBAM and IIS)
 1. Using the MBAM Configurator; remove the Admin and Self Service Portals from  the IIS server
 2. Install MBAM 2.5 SP1
 3. Do not configure it at this time  
-4. Install the May 2019 Rollup on the IIS server(https://www.microsoft.com/download/details.aspx?id=58345)
-5. Using the MBAM Configurator; re-add the Admin and Self Service Portals to the IIS server 
-6. This will configure the sites using the latest MBAM code from the May 2019 Rollup
-7. Open an elevated command prompt, Type: **IISRESET** and Hit Enter.
+4. Using the MBAM Configurator; re-add the Admin and Self Service Portals to the IIS server 
+5. Open an elevated command prompt, type: **IISRESET** and hit Enter.
```
Comment (JohanFreelancer9) [inline mdop/mbam-v25/upgrading-to-mbam-25-sp1-from-mbam-25.md]:
```suggestion
5. Open an elevated command prompt, type **IISRESET**, and hit Enter.
```

File: mdop/mbam-v25/upgrading-to-mbam-25-sp1-from-mbam-25.md [hunk 9]
```diff
@@ -29,21 +29,17 @@ Verify you have a current documentation of your MBAM environment, including all
    Note: You will not see an option to remove the Databases; this is expected.
```
Comment (JohanFreelancer9) [inline mdop/mbam-v25/upgrading-to-mbam-25-sp1-from-mbam-25.md]:
```suggestion
  > [!NOTE]
  > You will not see an option to remove the Databases; this is expected.  
```

File: mdop/mbam-v25/upgrading-to-mbam-25-sp1-from-mbam-25.md [hunk 10]
```diff
@@ -29,21 +29,17 @@ Verify you have a current documentation of your MBAM environment, including all
...
 2. Install 2.5 SP1 (Located with MDOP - Microsoft Desktop Optimization Pack 2015 from the Volume Licensing Service Center site:  <https://www.microsoft.com/Licensing/servicecenter/default.aspx>
 3. Do not configure it at this time 
-4. Install the May 2019 Rollup: https://www.microsoft.com/download/details.aspx?id=58345
-5. Using the MBAM Configurator; re-add the Reports role
-6. This will configure the SSRS connection using the latest MBAM code from the rollup 
-7. Using the MBAM Configurator; re-add the SQL Database role on the SQL Server.
-8. At the end, you will be warned that the DBs already exist and  weren’t created, but this is expected.
-9. This process updates the existing databases to the current version  being installed                  
+4. Using the MBAM Configurator; re-add the Reports role
+5. Using the MBAM Configurator; re-add the SQL Database role on the SQL Server.
```
Comment (JohanFreelancer9) [inline mdop/mbam-v25/upgrading-to-mbam-25-sp1-from-mbam-25.md]:
```suggestion
5. Using the MBAM Configurator; re-add the SQL Database role on the SQL Server
```

File: mdop/mbam-v25/upgrading-to-mbam-25-sp1-from-mbam-25.md [hunk 11]
```diff
@@ -29,21 +29,17 @@ Verify you have a current documentation of your MBAM environment, including all
...
+6. At the end, you will be warned that the DBs already exist and  weren’t created, but this is expected.
```
Comment (JohanFreelancer9) [inline mdop/mbam-v25/upgrading-to-mbam-25-sp1-from-mbam-25.md]:
```suggestion
6. At the end, you will be warned that the DBs already exist and  weren’t created, but this is expected
```

**Comment by mypil** [comment]
@MaratMussabekov - please merge the commit on the copy/edit review done by JohanFreelancer9 or make the necessary changes on the PR following the suggestions made. Thank you.

File: mdop/mbam-v25/upgrading-to-mbam-25-sp1-from-mbam-25.md (hunk 11 above)
Comment (illfated) [reply mdop/mbam-v25/upgrading-to-mbam-25-sp1-from-mbam-25.md]: cosmetic: duplicate space after "and"

**Comment by mypil** [comment]
@Dansimp - PR has been copy edited and is ready for final review. This is based on issue #5417.

Thank you.

cc: @kenwith

**Comment by illfated** [comment]
#sign-off

**Pull request merged**